app/models/subscriber.rb

```diff
@@ -0,0 +1,32 @@
+class Subscriber
```

Comment:
I don't agree with definition of models, but let's go with it.

I also have come to dislike the name services, and jobs has a different connotation (background jobs). I really don't have a better name for this than object... It has no special property I can think of. It's just a PORO.